README.md

```diff
@@ -186,31 +196,31 @@ By the end of this lesson. You should be able to set up two separate servers tha
 
 * **Question:** Describe what is happening in the code above.
 
-* **Your Answer:**
+* **Your Answer: it is a test to see if currentuser has a value, if it does, goes to the user page, if not it redirects the user to the login page
 
 ---
 
 - [ ] Now try logging in. Then, when you're on the `/users` page, refresh the page.
 
 * **Question:** What happens and why?
 
-* **Your Answer:**
+* **Your Answer: you get redirested to the login page, because the refresh has not finished updating the currentuser before the app hit the redirect code, so as far as the app is concerned you are not logged in
 
 ---
 
 - [ ] To solve this problem, let's add a `loading` key to our App's state, with the default value set to `true`. When `componentDidMount()` finishes, set the `loading` key to equal `false`. Using this key, solve the issue of refreshing on the `/users` page. Make sure everyting continues to work whether you are logged in or out.
 
 * **Question:** What did you do to solve this problem?
 
-* **Your Answer:**
+* **Your Answer: this stops the currentuser check until the call is finished, allowing the currentuser to get set before a check is made against it
 
 ---
 
 - [ ] We will have the same problem on the `/users/<userId>/posts` page. Use the same strategy to have this page load correctly on refresh.
 
 * **Question:** In what component did you add the `loading` property and why?
 
-* **Your Answer:**
```

Comment:
Adding it to users should keep the app from rendering until there are posts to display.